app/assets/stylesheets/components/_callout-box.scss

```diff
@@ -33,6 +33,12 @@
     border-right: 8px solid darken($ocean-blue, 10%);
     box-shadow: -1.5em 0 0 0 $ocean-blue;
   }
+
+  .slate-grey {
+    background-color: $slate-grey;
+    border-right: 8px solid darken($slate-grey, 10%);
```

Comment:
wonder if this is named appropriately, it has a box shadow, border and color..

Reply:
do you mean the class name? If you expand above, I'm following the other classes used for the callout box titles.

Reply:
I see than yeah this makes sense now